src/Acme/DemoBundle/Controller/SecuredController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Acme\DemoBundle\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\SecurityContext;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+
+/**
+ * @Route("/demo/secured")
+ */
+class SecuredController extends Controller
+{
+    /**
+     * @Route("/login", name="_demo_login")
+     * @Template()
+     */
+    public function loginAction(Request $request)
+    {
+        if ($request->attributes->has(SecurityContext::AUTHENTICATION_ERROR)) {
+            $error = $request->attributes->get(SecurityContext::AUTHENTICATION_ERROR);
+        } else {
```

Comment:
__[development][phpmd]__
* The method loginAction uses an else expression. Else is never necessary and you can simplify the code to work without else.